### PR TITLE
[KOGITO-162] - Migrate the Kogito base images from centos to ubi8

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: Filippe Spolti <fspolti@redhat.com>

### CODE REVIEW

**Comment by spolti** [comment]
@mswiderski SHould the GraalVM be updated to 19.1.1?

**Comment by spolti** [comment]
Just tested https://github.com/kiegroup/kogito-examples/tree/0.2.0/drools-quarkus-example on openshift with the new images, works as expected.

**Comment by ricardozanini** [comment]
@spolti we have a JIRA to upgrade to 19.1.1. But once we upgrade the Kogito runtimes to a compatible quarkus version, which is 0.20:

https://issues.jboss.org/browse/KOGITO-98

Feel free to assign this JIRA to you and work on it. Many thanks!

**Review by ricardozanini — CHANGES_REQUESTED** [review]
~The ImageStream also need to be updated?~

Just found the file, lgtm.

**Review by ricardozanini — APPROVED** [review]

**Comment by mswiderski** [comment]
Have you tested onboarding example? Reason why i as is that is used https and by that some native library from GraalVM so would be good to check that UBI 8 has all libraries needed to make the https based connections. 

When it comes to graalvm version - I will try to do upgrade of quarkus version soon and then we align graalvm with it. Will let you know

**Comment by ricardozanini** [comment]
@mswiderski thanks for noticing the TLS thing. This is really a PITA, but thankfully they made some changes on 19.1.1/Quarkus 0.20 to avoid the issues we're having.

**Comment by spolti** [comment]
@mswiderski not yet, thanks for remind me, will test it before remove the wip label.
@ricardozanini cool, I can take deal with the graalvm update. 

Thanks.

File: s2i/modules/add-kogito-user/module.yaml [hunk 1]
```diff
@@ -1,8 +1,14 @@
 schema_version: 1
+version: "0.3.0"
+
```
Comment (ricardozanini) [inline s2i/modules/add-kogito-user/module.yaml]: Can we remove this new line?

File: s2i/modules/add-kogito-user/module.yaml [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
 execute:
 - script: add-user
 
 envs:
 - name: "KOGITO_HOME"
   value: "/home/kogito"
+- name: "USER"
+  value: "kogito"
+
+
```
Comment (ricardozanini) [inline s2i/modules/add-kogito-user/module.yaml]: double new line

**Review by ricardozanini — CHANGES_REQUESTED** [review]
Micro things. But I guess we should coordinate this merge with https://issues.jboss.org/browse/KOGITO-98

**Comment by spolti** [comment]
@mswiderski @ricardozanini just tried the onboarding example, work as expected.

**Review by mswiderski — APPROVED** [review]
looks good to me, if we merge the upgrade of quarkus (https://github.com/kiegroup/kogito-bom/pull/48) then we should upgrade graalvm here as well.

**Comment by ricardozanini** [comment]
@mswiderski we already upgraded to GraalVM 19.1.x and will merge this soon to run e2e tests with https://github.com/kiegroup/kogito-bom/pull/48. If you decide to not merge for 0.3.0, we can rollback this upgrade.
Sorry, I didn't see your comment before.

**Comment by spolti** [comment]
This PR is ready.

**Pull request merged**